### PR TITLE
container insights 3.1.10 release

### PR DESCRIPTION
**Pull request by wanlonghenry**
container insights 3.1.10 release

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -8,6 +8,38 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
...
+- Windows
+  - Ruby - 3.1.1
+  - Fluent-bit - 2.0.14
+  - Telegraf - 1.24.2
+  - Fluentd - 1.14.6
+  - Windows AMA - 46.3.2
+  - Golang - 1.20.3
+##### Code change log
+- Common
+  - Add Containerlogv2 support in dataCollectionSettings
```
Comment (jatakiajanvi12) [inline ReleaseNotes.md:32]: I think Amol mentioned some tool to update the change log. Were you able to try that?

Comment (wanlonghenry) [reply ReleaseNotes.md:32]: good point  let me try that

Comment (wanlonghenry) [reply ReleaseNotes.md:32]: The generated release notes will be something like this, what you guys think, we can definitely include this in our tag management though (eg. after merge, create 3.1.10 tag with those message): ## What's Changed
* Gangams/fix typo envvar name by @ganga1980 in https://github.com/microsoft/Docker-Provider/pull/995
* Fix weird non-space whitespace by @TeamDman in https://github.com/microsoft/Docker-Provider/pull/996
* Troubleshoot doc update for MSI GA by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/991
* fix fluentd crash in windows by @ganga1980 in https://github.com/microsoft/Docker-Provider/pull/1001
* Longw/window vul by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/998
* Longw/containerlogv2 dcr  by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/994
* Windows spin-wait thread pool issue by @jatakiajanvi12 in https://github.com/microsoft/Docker-Provider/pull/1002
* Longw/containerlogv2 dcr merge by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/1004
* upgrade fluent-bit to 2.0.14 by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/1005

## New Contributors
* @TeamDman made their first contribution in https://github.com/microsoft/Docker-Provider/pull/996

**Full Changelog**: https://github.com/microsoft/Docker-Provider/compare/3.1.9...3.1.10-test

Comment (pfrcks) [reply ReleaseNotes.md:32]: yes we can add it in tag info as well as release notes

**Review by ganga1980 — APPROVED** [review]
lgtm

**Pull request merged**